dev/41c899fa-89b1-4d23-93a1-e03f6f5e88eb/device.lp

```diff
@@ -0,0 +1,19 @@
+(librepcb_device 41c899fa-89b1-4d23-93a1-e03f6f5e88eb
+ (name "SGP30")
+ (description "Gas sensor for indoor air quality applications.")
```

Comment:
Do you want to add a link to the datasheet in the description? https://www.sensirion.com/file/datasheet_sgp30/.